gmso/core/topology.py

```diff
@@ -369,11 +370,25 @@ def add_connection(self, connection, update_types=True):
         update_types : bool, default=True
             If True also add any Potential object associated with connection to the
             topology.
+
+        Returns
+        _______
+        Connection
```

Comment:
can you update the returns section a bit more?
```suggestion
        Connection : gmso.Connection
```

And for the other methods, specifying the `gmso.Bond` etc. would probably be nice.